views/Products/List/index.tsx

```diff
@@ -3,14 +3,21 @@ import { FC } from 'react'
 import ProductItem from '@/components/ProductItem'
 import { Product } from '@/services/GraphQL/products/types'
 import styles from '@/styles/Products.module.scss'
-const ProductList: FC<{ products: Product[] }> = ({ products }) => (
-  <div className={styles.container}>
-    <div className={styles['products-list']}>
-      {products.map((product: Product) => (
-        <ProductItem key={`product-${product.id}`} product={product} />
-      ))}
+
+interface ProductListProps {
+  products: Product[]
+}
+
+const ProductList: FC<ProductListProps> = ({ products }) => {
+  return (
+    <div className={styles.container}>
+      <div className={styles['products-list']}>
+        {products.map((product: Product) => (
+          <ProductItem key={`product-${product.id}`} product={product} />
+        ))}
+      </div>
     </div>
-  </div>
-)
+  )
+}
```

Comment:
I think this is a regression

@Eghinner please check

Reply:
@eduardo-quispe what about leaving as it was?

Reply:
updated